### ISSUE TITLE
How does the png format label image be converted to groundtruth

### ISSUE THREAD

**Issue by CheeseInTrap** [wkentaro/pytorch-fcn#94]
I am confused when I read the code, it seems to be just openning  the png format label image and converting it to numpy , then it becomes the groundtruth which can be used to calculate loss with the net predict result 'score',did I miss some important process ?

**Comment by wkentaro** [comment]
Softmax cross entropy uses label image (image filled with label values) as the ground truth, so opening png and converting to ndarray is enough.
https://github.com/wkentaro/pytorch-fcn/blob/f6766058ad7968721f6217c378dff32716ee9fc6/torchfcn/trainer.py#L20-L39
